src/lib/scheduler_clients/models.py

```diff
@@ -4,23 +4,92 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 # models
-from typing import List, Optional
-from lib.models import CamelModel
+from typing import Dict, List, Optional, Any
```

Comment:
This file is intended to contain the generic or "abstracted" model. 
The models here a re the same across scheduler but do not contain scheduler specific logic. 
Move the scheduler specific logic into:

- src/lib/scheduler_clients/slurm/models.py
- src/lib/scheduler_clients/pbs/models.py